torchci/pages/api/flaky-tests/disable.ts

```diff
@@ -57,40 +62,42 @@ export async function handleFlakyTest(
   if (matchingIssues.length !== 0) {
     // There is a matching issue
     const matchingIssue = matchingIssues[0];
-    if (matchingIssue.state === "open") {
-      const body = `Another case of trunk flakiness has been found [here](${getLatestTrunkJobURL(
-        test
-      )}).
+    if (wasRecent(test)) {
```

Comment:
Nit: The diff looks a bit shorter here is you just short circuiting `wasRecent` check I think, i.e.

```
if (not wasRecent(test)) {
  return ;
}
```

Reply:
done